tests/integration/security/healthcheck/mtls_healthcheck_test.go

```diff
@@ -34,8 +35,13 @@ var (
 )
 
 func TestMain(m *testing.M) {
+	// We only run this test in postsubmit since that runs on prow.
+	// Presubmit runs on CircleCI, using a Minikube environment, which seems
+	// causing healthcheck app deployment timeout.
+	// For more details, see https://github.com/istio/istio/issues/12754.
 	framework.NewSuite("mtls_healthcheck", m).
 		RequireEnvironment(environment.Kube).
+		Label(label.Postsubmit).
```

Comment:
The Postsubmit label hasn't been wired yet. Just removing Skip below should be enough to get the test running in the queues (non-required presubmit & postsubmit) again.

Reply:
Done.

Reply:
I believe this will make test run in prow/circleci-integ-k8s since circleci currently it's empty...